ir/pattern/conjunction.rs

```diff
@@ -124,13 +124,15 @@ impl Conjunction {
     }
 
     pub(crate) fn find_disjoint(&self, block_context: &BlockContext) -> ControlFlow<(Variable, Option<Span>)> {
```

Comment:
What is the API doing? It returns a single Continue<(Variable, Span)>? isn't it possible to have more than 1 disjoint variable?

Reply:
The `ControlFlow<B>` `B` generic is the `Break` type. It returns `Break` with the first found pair, or `Continue` if nothing found.
We could return all disjoint variables here too, though we can only report one to the user anyway.